src/App.jsx

```diff
@@ -1,43 +1,138 @@
 import React from 'react';
```

Comment:
A lot of these changes look like changes that @Dave-Gallup made a PR for and I just merged. Please rebase from the dashboard-spa dev branch.